src/ontology/mondo-edit.obo

```diff
@@ -430439,6 +430582,410 @@ property_value: IAO:0000231 "out of scope" xsd:string
+subset: ordo_disorder
+synonym: "CARST" EXACT [Orphanet:566192]
+xref: ICD10:D69.4 {source="Orphanet:566192", source="MONDO:mondoSubClassOfSource"}
+xref: Orphanet:566192 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018796 {source="https://orcid.org/0000-0001-5208-3432"} ! isolated constitutional thrombocytopenia
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/4036 ! FYB1
+
+[Term]
+id: MONDO:0035449
+name: atelencephaly
+subset: ordo_subtype_of_a_disorder
+synonym: "atelencephalic microcephaly" EXACT [Orphanet:566852]
+xref: Orphanet:566852 {source="MONDO:equivalentTo"}
+is_a: MONDO:0016054 {source="https://orcid.org/0000-0001-5208-3432"} ! cerebral malformation
+
+[Term]
+id: MONDO:0035450
+name: aprosencephaly
+subset: ordo_subtype_of_a_disorder
+xref: Orphanet:566857 {source="MONDO:equivalentTo"}
+is_a: MONDO:0016054 {source="https://orcid.org/0000-0001-5208-3432"} ! cerebral malformation
+
+[Term]
+id: MONDO:0035452
+name: mueller-weiss syndrome
+def: "A rare bone disease characterized by spontaneous adult-onset tarsal navicular osteonecrosis. Patients present with chronic mid- and hindfoot pain, swelling and tenderness over the dorsomedial aspect of the midfoot, flattening of the medial longitudinal arch, and pes planovarus. Radiographic findings include comma-shaped deformity due to collapse of the lateral part of the navicular bone and medial or dorsal protrusion of a portion or the entire bone. The condition may be bilateral or asymmetric and associated with pathological fractures." [Orphanet:566943]
+subset: ordo_disorder
+synonym: "Brailsford disease" EXACT [Orphanet:566943]
+synonym: "Mueller-Weiss osteonecrosis of the tarsal bone" EXACT [Orphanet:566943]
+xref: Orphanet:566943 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005381 {source="https://orcid.org/0000-0001-5208-3432"} ! bone disorder
+
+[Term]
+id: MONDO:0035459
+name: idiopathic multidrug-resistant nephrotic syndrome
+subset: ordo_subtype_of_a_disorder
+xref: ICD10:N04.8 {source="MONDO:mondoSubClassOfSource", source="Orphanet:567550"}
+xref: Orphanet:567550 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005377 {source="https://orcid.org/0000-0001-5208-3432"} ! nephrotic syndrome
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/18143
+
+[Term]
+id: MONDO:0035460
+name: idiopathic steroid-resistant nephrotic syndrome with sensitivity to second-line immunosuppressive therapy
+subset: ordo_subtype_of_a_disorder
+synonym: "idiopathic steroid-resistant nephrotic syndrome with sensitivity to intensified immunosuppression" EXACT [Orphanet:567552]
+xref: ICD10:N04.8 {source="MONDO:mondoSubClassOfSource", source="Orphanet:567552"}
+xref: Orphanet:567552 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005377 {source="https://orcid.org/0000-0001-5208-3432"} ! nephrotic syndrome
```

Comment:
should be more precise: idiopathic nephrotic syndrome

Reply:
agree